src/Network/Probe.php

```diff
@@ -565,8 +565,9 @@ public static function getWebfingerArray(string $uri): array
 			$nick = '';
 			$addr = '';
 
-			$path_parts = explode('/', trim($parts['path'] ?? '', '/'));
-			if (is_array($path_parts)) {
+			if (array_key_exists('path', $parts) && strval($parts['path']) !== '') {
```

Comment:
You also have to account for the `trim()` call in your condition.
```suggestion
			if (array_key_exists('path', $parts) && trim(strval($parts['path']), '/') !== '') {
```